macros/schedules.sql

```diff
@@ -0,0 +1,51 @@
+{%- macro get_nth_weekday(nth_weekday) -%}
+    case {{ nth_weekday }}
+        when -1 then 'last ' when 1 then 'first ' when 2 then 'second '
+        when 3 then 'third ' when 4 then 'fourth ' else ''
```

Comment:
For ease of reading each condition.
```suggestion
        when -1 then 'last '
        when 1 then 'first '
        when 2 then 'second '
        when 3 then 'third '
        when 4 then 'fourth ' 
        else ''
```